src/gatsby/plugins/gatsby-plugin-openapi/gatsby-node.ts

```diff
@@ -184,6 +186,46 @@ export const onCreateNode = async ({
         parent: node,
         child: paramNode,
       });
-    })
+    });
+
+    const bodyParameterSchemaString =
+      node.path.requestBody !== null &&
+      node.path.requestBody.content !== null &&
+      node.path.requestBody.content.schema;
```

Comment:
I think we can clean this up with optional chaining.

Reply:
I agree, but I actually reverted this from optional chaining specifically because it breaks something it the Gatsby Build process 👀 that only applies to this file.

Reply:
Here's what happened to the Vercel preview when I used optional chaining 
![image](https://user-images.githubusercontent.com/35509934/130252511-300215a9-6d68-4a35-80c5-16af43f79e55.png)

Reply:
@leeandher I see it used here: https://github.com/getsentry/sentry-docs/blob/master/src/components/codeContext.tsx#L130

Reply:
@NisanthanNanthakumar Yeah, it seems to only break in the `src/gatsby/plugins/gatsby-plugin-openapi/gatsby-node.ts` file, because Gatsby processes those as is, without webpack. Outside of that it seems to work just fine.

See: https://github.com/gatsbyjs/gatsby/issues/23802

Reply:
hmm the values could also be `undefined`.

```suggestion
      node.path.requestBody &&
      node.path.requestBody.content &&
      node.path.requestBody.content.schema;
```

Reply:
Added a falsy check instead, thanks :)